package.json

```diff
@@ -30,7 +30,7 @@
     "start": "npm-run-all build run",
     "lint": "npm-run-all lint:*",
     "lint:addon": "npm run package && addons-linter ./addon.xpi -o text",
-    "lint:js": "eslint addon",
+    "lint:js": "eslint .",
```

Comment:
This change causes a failure for me because there's a file in ./build that doesn't pass lint. Maybe it should be `eslint addon experiment` instead?

Reply:
Odd, I ran `npm run build` and it still passes linting.
Are you seeing an error in ./addon/build? I don't see a root ./build directory locally.

Reply:
@pdehaan Ok the build directory may have been old garbage left over. But, if I do "npm run package" then I get a similar problem where lint chokes on a file left over in web-ext-artifacts.

Reply:
Added "web-ext-artifacts" to the .eslintignore. I thought that folder only had .ZIP and .XPI files, but maybe something got unzipped at some point. 
Not sure if we should be doing an `rm -rf` on the ./addon/build directory as part of the build step so we know that the directory is always clean and doesn't collect lingering files. Pretty sure my addon/build folder still has a few old JS and JSON files from when I was debugging something a long time ago.